src/pages/[platform]/build-a-backend/data/connect-event-api/index.mdx

```diff
@@ -0,0 +1,100 @@
+import { getCustomStaticPath } from '@/utils/getCustomStaticPath';
+
+export const meta = {
+  title: 'Connect to AWS AppSync Events',
```

Comment:
Make this title: "Real time events"

Reply:
we have a separate page with that verbiage https://docs.amplify.aws/react/build-a-backend/data/subscribe-data/

Reply:
What about Pub/Sub events?